.github/workflows/publish_python_package.yml

```diff
@@ -0,0 +1,43 @@
+name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+
+on:
+  push:
+    branches:
+      - feature-publish-pypi
+    paths:
+      - 'drafttopic/about.py'
+
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.7"
+      - name: Install pypa/build
+        run: >-
+          python -m pip install build --user
+      - name: Build a binary wheel and a source tarball
+        run: >-
+          python -m
+          build
+          --sdist
+          --wheel
+          --outdir dist/
+          .
+
+      - name: Publish distribution 📦 to Test PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          password: ${{ secrets.ISARANTO_TEST_TOKEN }}
+          repository_url: https://test.pypi.org/legacy/
+
+    #    - name: Publish distribution 📦 to PyPI
+    #      if: startsWith(github.ref, 'refs/tags')
```

Comment:
In order to push on tag creation we will have to create another action that pushes the tags once a commit is pushed to master. That action needs to be triggered with a PAT other than `GITHUB_TOKEN` as the github actions user does not trigger additional actions in order to avoid circular executon.